manager/manager.go

```diff
@@ -1145,3 +1156,24 @@ func newIngressNetwork() *api.Network {
 		},
 	}
 }
+
+func newPredefinedNetwork(name, driver string) *api.Network {
+	return &api.Network{
+		ID: identity.NewID(),
+		Spec: api.NetworkSpec{
+			Annotations: api.Annotations{
+				Name: name,
+				Labels: map[string]string{
+					networkallocator.PredefinedNetworkLabel: "true",
+				},
+			},
+			DriverConfig: &api.Driver{Name: driver},
+			IPAM: &api.IPAMOptions{
+				Driver: &api.Driver{},
+				Configs: []*api.IPAMConfig{
+					{},
```

Comment:
I kind of followed what was being done for the ingress network. I will check whether it is required.